README.md

```diff
@@ -16,18 +16,29 @@ downstream repos.
 
 ## Configuration
 
-1. Create a `robo.yml` file in the root of your codebase. `robo.drupal.example.yml`
+- Create a `robo.yml` file in the root of your codebase. `robo.drupal.example.yml`
```

Comment:
The order didn't seem like it mattered for these, so I changed them. I can be talked into reverting that though.